modules/sorts/test/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_executable(stest test.cpp)
+target_link_libraries(stest Sorts)
+
+# Tests
+add_test(
+    NAME "example test"
+    COMMAND stest
```

Comment:
Note that to produce a test report that's readable by Jenkins (the continuous integration system we're using) you need additional arguments. These were never added to the test guide; I've updated it to include them. You will also need the boost.xsl file (which I just added to the repo, so you'll have to fetch and rebase) which Jenkins uses to parse the result.